frontend/src/routes/Governance/discovered/ByCluster/common.tsx

```diff
@@ -108,6 +108,7 @@ export const byClusterCols = (
   subscriptions: Subscription[],
   channels: Channel[],
   policyKind: string,
+  diabledSeverityTooltip: boolean,
```

Comment:
It'd be nice to fix the typo in the variable name eventually `diabledSeverityTooltip` => `disabledSeverityTooltip`